test/integration/features/cmd-config.feature

```diff
@@ -174,10 +174,17 @@ user defined options which changes default behaviour of Minishift.
     | property        | value                                                                                            | expected                                                                                        |
     | iso-url         | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso        | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso       |
     | iso-url         | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso  | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso |
-    | iso-url         | file:///home/me/Downloads/my_handmade_b2d.iso                                                    | file:///home/me/Downloads/my_handmade_b2d.iso                                                   |
     | iso-url         | b2d                                                                                              | b2d                                                                                             |
     | iso-url         | centos                                                                                           | centos                                                                                          |
-
+       
+  @quick
+  Scenario: Setting iso-url key to a non exsisting path
+     When executing "minishift config set iso-url file:///home/Downloads/my_handmade_centos.iso" fails
+     Then stderr should contain
+      """
+      'file:///home/Downloads/my_handmade_centos.iso' file is not present
```

Comment:
@anjannath I have tried that locally and the error is in brackets, it seems strange, why is it that way?

```
[agajdosi@dhcp-24-156 ~]$ minishift config set iso-url file:///home/agajdosi/my.iso
['file:///home/agajdosi/my.iso' file is not present]
```

Reply:
@agajdosi I think we are printing a string slice, so the brackets appear, will have to check the code.

Reply:
@anjannath Yeah, it passes fine, I was just curious.

Reply:
@agajdosi Hmm, it is because its a slice.. we append all the errors to an error slice while running validation checks, and return that slice to `config` command which it then prints using `Error()`

Reply:
@anjannath Ok, thank you.